src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja

```diff
@@ -190,181 +227,108 @@
+			})
+			newRecipient.addEventListener('address-input', (event) => {
+				validateForm();
+			})
+			newRecipient.addEventListener('unit-change', (event) => {
+				validateForm();
+			})
+			newRecipient.addEventListener('send-max', (event) => {
+				setMaxAmount(newRecipient.recipientId);
+			})
+
+				
+
+
 			recipients.appendChild(newRecipient);
-			document.getElementById('address-scan-' + i).addEventListener('scan', e=>{
-				let addr = e.detail.result;
-				if(addr == null){
-					return;
-				}
-				// remove bitcoin: stuff
-				if(addr.indexOf("bitcoin:") >= 0){
-					addr = addr.substr(addr.indexOf("bitcoin:")+8);
-				}
-				let arr = addr.split("?");
-				addr = arr[0];
-				document.getElementById("address_" + i).value = addr;
-				let evt = new Event('input');
-				document.getElementById("address_" + i).dispatchEvent(evt);
-				// parse metadata like amount and message
-				if(arr.length > 1){
-					arr = arr[1].split("&");
-					arr.forEach((e)=>{
-						if(e.startsWith("amount=")){
-							let val = parseFloat(e.substr(7));
-							if(units[i] == 'sat'){
-								val = Math.round(val*1e8);
-							}
-							document.getElementById("amount_" + i).value = val;
-							let evt = new Event('input');
-							document.getElementById("amount_" + i).dispatchEvent(evt);
-						}
-						if(e.startsWith("message=") || e.startsWith("label=")){
-							document.getElementById("label_" + i).value = e.split("=")[1];
-						}
-					});
-				}
-			});
-			if (amounts.length == 1) {
-				document.getElementById('remove-recipient').style.display = 'none';
-				document.getElementById('coin-selection-row').style['margin-top'] = '30px';
+
+			// This adds the id's to the subtract_from_recipient_id_select
+			var feeSelectionComponent = document.getElementById('fee-selection-component');
+			var subtractFromRecipientIdSelect = feeSelectionComponent.subtractFrom.children.subtract_from_recipient_id_select;
+			var opt = document.createElement('option');
+			opt.value = recipientId;
+			opt.innerHTML = newRecipient.title;
+			subtractFromRecipientIdSelect.appendChild(opt);		
+
+
+			// Make the remove button visible again for all recipient components
+			setVisibilityRemoveRecipientButton();
+
+			// control visibility  and vertical spaces
+			if (getRecipientIds().length == 1) {
+				FeeSelectionComponent.showSubtractFrom(false);
+				document.getElementById('vertical-space-for-subtract-fees-from-amount-checkbox').style['margin-top'] = '0px';
 			} else {
-				document.getElementById('remove-recipient').style.display = 'block';
-				document.getElementById('coin-selection-row').style['margin-top'] = '90px';
+				document.getElementById('vertical-space-for-subtract-fees-from-amount-checkbox').style['margin-top'] = '60px';
 				if (document.getElementById('subtract').checked) {
 					FeeSelectionComponent.showSubtractFrom(true)
 				}
 			}
-			calculateConvertedUnit(i);
+
+			document.getElementById(`recipient_${recipientId}`).calculateConvertedUnit();
```

Comment:
Why is this needed?

Reply:
It is not needed. Thanks for pointing that out.